etc/lexemes.tsv

```diff
@@ -1,2 +1,3 @@
 LEXEME	REPLACEMENT	COMMENT
 t̪oŋkaŋmala || ̃ai za	t̪oŋkaŋmala	
+zimbu || ̃ai za	zimbu	
```

Comment:
what's happening here?

Reply:
They have `| |` as a delimiter of two entries or a column wrongly written in wikipedia pull (this was taken from there), so I corrected this manually by taking only the first entry. But it may be useful to check in fact if the conversion from wikipedia is failsave !

Reply:
But given that I would opt for not calling the download command that often, it could be handled in a separate PR.

Reply:
Ok, I was reading that as "replace "zimbu" with "ai za zimbu".